src/legacy/core_plugins/kibana/public/dashboard/panel/dashboard_panel.tsx

```diff
@@ -17,127 +17,162 @@
  * under the License.
  */
 
-import React from 'react';
-import PropTypes from 'prop-types';
-import { injectI18n, FormattedMessage } from '@kbn/i18n/react';
+import { FormattedMessage, injectI18n } from '@kbn/i18n/react';
+
+import { EuiPanel } from '@elastic/eui';
 import classNames from 'classnames';
 import _ from 'lodash';
-
-import { PanelHeader } from './panel_header';
+import React from 'react';
+import {
+  ContainerState,
+  Embeddable,
+  EmbeddableFactory,
+  EmbeddableMetadata,
+  EmbeddableState,
+} from 'ui/embeddable';
+import { EmbeddableActions, EmbeddableErrorAction } from '../actions';
+import { PanelState } from '../selectors';
 import { PanelError } from './panel_error';
+import { PanelHeader } from './panel_header/panel_header';
+
+interface DashboardPanelProps {
+  intl: any;
+  viewOnlyMode: boolean;
+  onPanelFocused: (panel: string) => {};
+  onPanelBlurred: (panel: string) => {};
+  error: string | object;
+  destroy: () => void;
+  containerState: ContainerState;
+  embeddableFactory: EmbeddableFactory;
+  embeddableStateChanged: (embeddableStateChanges: EmbeddableState) => {};
+  embeddableIsInitializing: EmbeddableActions;
+  embeddableIsInitialized: (embeddableIsInitialized: EmbeddableMetadata) => void;
+  embeddableError: EmbeddableErrorAction;
+  initialized: boolean;
+  panel: PanelState;
+  isPopoverOpen: boolean;
```

Comment:
this one also looks unused

Reply:
Awesome - removed embeddableError and isPopoverOpen. The panel and initialized cause errors when I remove them here so I left them.